.github/workflows/ci.yml

```diff
@@ -41,18 +41,16 @@ jobs:
       matrix:
         os:
           - 'amazonlinux-2'
-          - 'centos-7'
```

Comment:
Why are you removing EL7?

Reply:
Source build doesn't work on centos7 and I don't have time to fix it right now.

Reply:
There's a trade off, test source builds or have centos 7 in the mix.  Is there a way to exclude a particular suite, os combination from the standard tests?

Reply:
You can exclude per suite.

Reply:
Any idea about the syntax for doing the exclude?  The ci.yaml code generates kitchen test xxxxx for the cross product of the suite and os and blows up if put an exclude in a suite in the kitchen.yml file?